lib/ruby_lsp/requests/support/sorbet.rb

```diff
@@ -0,0 +1,158 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Requests
+    module Support
+      class Sorbet
+        class << self
+          extend T::Sig
+
+          ANNOTATIONS = T.let(
+            {
+              "abstract!" => Annotation.new(arity: 0, block: Annotation::Block::Unsupported),
```

Comment:
How was this list generated? Is there a way to ensure that we didn't miss anything?

Reply:
We did skip some things intentionally (mostly around T.proc.bind etc) and went with just the "top level" things.

Finding an automated way (and then deciding which we want and which we don't want) might be a good thing but I have no idea where to start and how much it really is needed (from a time spent vs benefit gained perspective)